docs/remote_rendering_using_docker.md

```diff
@@ -38,24 +38,33 @@ Instruct headless Chrome whether to ignore HTTPS errors during navigation. Per d
 Due to the security risk it's not recommended to ignore HTTPS errors.
 
 ```bash
-export IGNORE_HTTPS_ERRORS=true
+IGNORE_HTTPS_ERRORS=true
 ```
 
 **Enable Prometheus metrics:**
 
 You can enable [Prometheus](https://prometheus.io/) metrics endpoint `/metrics` using the environment variable `ENABLE_METRICS`. Node.js and render request duration metrics are included, see [output example](#prometheus-metrics-endpoint-output-example) for details.
 
 ```bash
-export ENABLE_METRICS=true
+ENABLE_METRICS=true
 ```
 
 **Log level:**
 
 Change the log level. Default is `info` and will include log messages with level `error`, `warning` and info.
 
+```bash
+LOG_LEVEL=debug
+```
+
+**Verbose logging:**
+
+Enable capturing verbose information when headless Chrome browses a page and takes a screenshot of it. It will capture all requests initiated and finished, when page is closed and any non-error logged to the console. These are logged as debug messages. Default is `false` and will only capture  errors when page crashes, uncaught exception thrown in page, page request fails or error logged to the console.
+
+Note that you need to change log level to `debug` for the verbose output to be seen in the logs.
```

Comment:
No, it''s not. Because when installed and running image renderer as plugin the plugin forwards all logs/levels. Grafana then decides based on its configured log level what to output. But good catch we probably need to add something regarding this in readme :+1: